students/lisa_ferrier/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,62 @@
+# slicing_lab.py
+# Lisa Ferrier, Python 210: Lesson 03 Exercise
+
+def exchange_first_last(seq):
+    '''This function exchanges the first and last items in a sequence '''
+    seq2 = seq[-1:]+seq[1:-1]+seq[:1]
```

Comment:
These should be snake_case, meaningful names, just like function names in Python